Commands/Search.cs

```diff
@@ -0,0 +1,111 @@
+﻿
+//System
+using System;
+using System.Threading.Tasks;
+
+//Discord 
+using Discord.Commands;
+
+//Google 
+using static Google.Apis.Customsearch.v1.CseResource;
+using static Google.Apis.Customsearch.v1.CseResource.ListRequest;
+using Google.Apis.Services;
+using Google.Apis.Customsearch.v1;
+using Discord;
+
+namespace SteamDiscordBot.Commands
+{
+    public class Search : ModuleBase
+    {
+        static SafeEnum SafeSearch;
+        static int Search_Number;
```

Comment:
Above variable doesn't have the concat of `_`, can we just make this SearchNumber